torch/quantization/fx/quantization_patterns.py

```diff
@@ -1083,8 +1084,10 @@ def should_insert_observer_for_output(
 
     def should_mark_output_quantized_from_input_quantized_status(
         self,
+        qconfig: QConfigAny
     ) -> bool:
-        return True
+        # FixQParamOps are the same as CopyNode in int8 quantization
+        return activation_dtype(qconfig) in [torch.quint8, torch.qint8]
```

Comment:
is this a behavior change? or is the end result the same and this was done elsewhere before?

Reply:
Yes, the end result is the same I think (no changes to test), but previous code probably works by coincidence I guess, since FixedQParams can't be treated as CopyNode when we do fp16 quantization.